client/src/app/pages/main-layout/main-layout.component.html

```diff
@@ -1,5 +1,5 @@
-<main class="h-screen overflow-hidden">
-  <div class="px-6 py-1.5 flex items-center justify-between">
+<main class="h-screen overflow-hidden flex flex-col">
+  <div class="px-6 py-1.5 flex items-center justify-between border-b-2 border-gray-200">
```

Comment:
This needs also a dark variant: `border-surface-300 dark:border-surface-700`, also i tried to unify the colors we used a bit and changed the 200 shades to 300 as it is only a small difference.